tests/test_clang_delta.py

```diff
@@ -32,3 +35,289 @@ async def equals_source(x):
+    cd = ClangDelta(find_clang_delta())
+    pump = clang_delta_pump(cd, "rename-fun")
+
+    async def is_interesting(x):
+        return True
+
+    problem = BasicReductionProblem(
+        ASSERTION_CRASHER, is_interesting, work=WorkContext(parallelism=1)
+    )
+
+    # Should not raise, should return the original
+    result = await pump(problem)
+    assert result == ASSERTION_CRASHER
+
+
+async def test_pump_handles_apply_error():
+    """Test clang_delta_pump handles ClangDeltaError from apply_transformation."""
+    cd = ClangDelta(find_clang_delta())
+    # Use rename-var which will error when trying to rename in simple code
+    pump = clang_delta_pump(cd, "rename-var")
+
+    async def is_interesting(x):
+        return True
+
+    # Simple code that has instances counted but fails during apply
+    source = b"int main() { return 0; }"
+    problem = BasicReductionProblem(
+        source, is_interesting, work=WorkContext(parallelism=1)
+    )
+
+    # Should not raise, should return the original
+    result = await pump(problem)
+    assert result == source
+
+
+# =============================================================================
+# Additional edge case tests for complete coverage
+# =============================================================================
+
+
+def test_find_clang_delta_when_found_in_path():
+    """Test find_clang_delta returns path when found via which()."""
+    from unittest.mock import patch
+
+    # Mock which to return a path
+    with patch(
+        "shrinkray.passes.clangdelta.which", return_value="/usr/bin/clang_delta"
+    ):
+        result = find_clang_delta()
+        assert result == "/usr/bin/clang_delta"
+
+
+def test_find_clang_delta_when_not_in_path():
+    """Test find_clang_delta falls back to glob when not in PATH."""
+    from unittest.mock import patch
+
+    # Mock which to return None (not in PATH)
+    with patch("shrinkray.passes.clangdelta.which", return_value=None):
+        # This will use glob to find clang_delta
+        result = find_clang_delta()
+        # Result depends on whether clang_delta is installed via homebrew/apt
+        # We just verify it doesn't crash and returns something (empty string if not found)
+        assert isinstance(result, str)
+
+
+def test_find_clang_delta_when_not_found_anywhere():
+    """Test find_clang_delta returns empty string when not found."""
+    from unittest.mock import patch
+
+    # Mock both which and glob to return nothing
+    with patch("shrinkray.passes.clangdelta.which", return_value=None):
+        with patch("shrinkray.passes.clangdelta.glob", return_value=[]):
+            result = find_clang_delta()
+            assert result == ""
+
+
+async def test_query_instances_raises_clang_delta_error():
+    """Test query_instances raises ClangDeltaError for non-assertion errors.
+
+    This tests line 133: raise ClangDeltaError(msg) when CalledProcessError
```

Comment:
Line 133 is not a durable reference, do you mean test_query_instances_on_crasher() ?